### PR TITLE
Added model removal helper functions

### PR DESCRIPTION
**Pull request by OWissett**
I have added some helper functions for removing models from a PDB. This is a common operation that I have been needing, so thought I would add it to the library.

I also fixed two of the tests which where failing due to issues with finding the PDB files, so they now use a path relative to the manifest to find the files, which should be more robust to files being moved around.

### CODE REVIEW

File: src/structs/pdb.rs [hunk 1]
```diff
@@ -998,7 +1044,35 @@ impl FromIterator<Model> for PDB {
...
+            }
+            let (pdb, _) = open_pdb(path.to_str().unwrap(), crate::StrictnessLevel::Loose).unwrap();
+
+            let model_count = pdb.model_count();
+            let mut test_pdb = pdb.clone();
+            test_pdb.remove_model(0);
+            assert_eq!(test_pdb.model_count(), model_count - 1);
+
+            let mut test_pdb = pdb.clone();
+            test_pdb.remove_all_models_except_first(); // calls remove_models_except, so no need to test that explicitly
```
Comment (douweschulte) [inline src/structs/pdb.rs:1070]: I do agree that it calls `remove_models_except` but I still think having a small test on a single structure removing a couple and leaving a handful others would be nice just to make sure the function does not break down in future refactorings.

**Review by douweschulte — CHANGES_REQUESTED** [review]
Looks really nice, seems like a very good addition to the suite of existing functions. I have some minor points however which you can find as comments.

**Comment by OWissett** [comment]
New commit addresses above comments

**Comment by douweschulte** [comment]
Congratulations! Nice addition I will merge it now.

**Pull request merged**